src/CakeSlayer/Db.hs

```diff
@@ -91,36 +101,76 @@ queryNamed
     -> [NamedParam]
     -> m [res]
 queryNamed q params =
-  withPool (\conn -> runExceptT $ Sql.queryNamed conn q params) >>= liftError
+    withPool (\conn -> runExceptT $ Sql.queryNamed conn q params) >>= liftError
 {-# INLINE queryNamed #-}
 
--- | Executes a query without arguments that is not expected to return results.
+{- | Performs a query with named parameters and custom 'RowParser' and returns a
+list of rows.
+-}
+queryWithNamed
```

Comment:
What about `queryWith`? Do you think it's not needed?

Reply:
Good idea to add them for consistency :+1: